RadarSDK/RadarUtils.m

```diff
@@ -105,6 +105,22 @@ + (BOOL)validLocation:(CLLocation *)location {
     return latitudeValid && longitudeValid && horizontalAccuracyValid;
 }
 
+#pragma mark - threading
++ (void)runOnMainThreadAsyncIfNecessary:(dispatch_block_t)block {
```

Comment:
since we're always invoking the callback on the main thread, should be fine to call this fn `runOnMainThreadAsync`

Reply:
These blocks could be invoked from either 1) main thread, or 2) the networking thread, or 3) the thread of `Radar` API callers. If they are invoked from the main thread, we don't want to do another main thread dispatch.